README.md

```diff
@@ -1,2 +1,71 @@
-# vue-cli-plugin-vuedock
-Docker plugin for the VueCLI
+# VueDock
+> Docker plugin for the Vue CLI 3
+
+[![Build Status](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock.svg?branch=master)](https://travis-ci.org/kaizendorks/vue-cli-plugin-vuedock)
+
+## Instructions
```

Comment:
Should we change this title to `Contributing` ?

Reply:
Yeah I actually added a todo at the end:  "Update readme and move this instructions somewhere else".

Note sure where a good place would be tho.

Reply:
Left it in the readme but added "Contributing", we can move somewhere else after if needed.